### ISSUE TITLE
certification archive lacks sha256shum line in json file

### ISSUE THREAD

**Issue by judovana** [adoptium/api.adoptium.net#434]
where
```
"package": {
                        "name": "java-17-openjdk-17.0.3.0.7-4.portable.jdk.el.x86_64.tar.xz",
                        "link": "https://openjdk-sources.osci.io/marketplace/17/java-17-openjdk-17.0.3.0.7-4.portable.jdk.el.x86_64.tar.xz",
                        "sha265sum": "47156f0eaba955602c9249054aa7e69a32e71de91d96359a0a3b672980d0f10d"
                    },
```
Allow marketplace to  identify change in LINK by changing sum, the 
```
 "aqavit_results_link": "https://openjdk-sources.osci.io/marketplace/17/java-17-openjdk-17.0.3.0.7-4.portable.jdk.el.x86_64.cert.tar.gz"
```

Is lacking this, and thus not allow the marektpalce to recognize the change in the aqavit_results_link, even allows to forge the results.

Can we have `aqavit_results_sha265sum`  field and appropriate logic, which allows updating the results if overwritten?

**Comment by smlambert** [comment]
Transferring to api repo, where I believe the change will need to take place to add this to the schema.

**Comment by smlambert** [comment]
@karianna - are you able to transfer this to https://github.com/adoptium/api.adoptium.net, as I appear not to have the correct permissions for the api repository to do so...

<img width="412" alt="Screen Shot 2022-05-27 at 5 44 36 AM" src="https://user-images.githubusercontent.com/2836948/170675219-de5ffb52-8f4c-4f0a-8dfd-c3ede284008f.png">

**Comment by smlambert** [comment]
There will also be associated documentation updates to be done in https://github.com/adoptium/website-v2/blob/main/src/asciidoc-pages/docs/aqavit-verification.adoc which I can take care of, upon the updated schema being available.

**Comment by tellison** [comment]
Agreed, and we'll need to add the checksum field to the schema and let publishers catch up before making it required. So expect multiple phases of implementation.
